client/src/domain/interview.tsx

```diff
@@ -52,6 +52,43 @@ export enum InterviewStatus {
   Canceled,
 }
 
+export enum InterviewResult {
+  Yes,
+  No,
+  Draft,
+}
+
+export function getSimpleInterviewResult(decision: Decision) {
```

Comment:
Why "simple"? Let's rename it `getSimpleInterviewResult` => `getInterviewResult`